### PR TITLE
Long path scenario

### PR DESCRIPTION
**Pull request by czepluch**
Scenario for https://github.com/raiden-network/raiden/issues/2758

### CODE REVIEW

**Comment by CosminNechifor** [comment]
Reviewing this

File: raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+            - assert: {from: 2, to: 1, total_deposit: 1000, balance: 1000, state: "opened"}
+      - serial:
+          name: "Make one transfer from 0 to 3"
+          tasks:
+            - transfer: {from: 0, to: 3, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 0 to 3"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 999, state: "opened"}
```
Comment (CosminNechifor) [inline raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml]: Why didn't you add the other asserts here, for channels like ``1<->2``

File: raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+            - assert: {from: 3, to: 2, total_deposit: 1000, balance: 1000, state: "opened"}
+      - serial:
+          name: "Make one transfer from 0 to 4"
+          tasks:
+            - transfer: {from: 0, to: 4, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 0 to 4"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 999, state: "opened"}
```
Comment (CosminNechifor) [inline raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml]: same here

File: raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml (hunk 1 above)
Comment (czepluch) [reply raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml]: I don't think that's necessary for this scenario. I think it's out of scope for it. I can add it if you really want, but I would just keep it as simple as possible.

**Pull request merged**